### PR TITLE
Align exercises.md and starter kit agendas

### PR DESCRIPTION
**Pull request by danfishgold**
Also: some markdown linting fixes

### CODE REVIEW

File: StarterKit/Session1/agenda.md [hunk 1]
```diff
@@ -89,9 +92,10 @@ Briefly discuss what it means to be important as a group (if you want to)
 ### (10 mins) Together - Discuss
 
 Discuss in the group:
+
 * lines covered by many people?
 * lines named but not by a lot of people
-* Agree less than 8 of the most important lines
+* Agree less than 10 of the most important lines
```
Comment (danfishgold) [inline StarterKit/Session1/agenda.md]: should this be 8 or 10?

File: StarterKit/Session2/agenda.md [hunk 2]
```diff
@@ -61,18 +61,21 @@ Reflect also on what kind of knowledge you used in this exercise:
 
 The goal of this exercise is to be a concrete thing to *do* when looking at new code for the first time. New code can be scary, doing something will help! Digital annotation can take a bit longer than on paper. Leave time for your group members to settle on a technique that works for them.
 
-### (10 mins) Independently - Examine structure
+### (12 mins) Independently - Examine structure
```
Comment (danfishgold) [inline StarterKit/Session2/agenda.md]: should this be 10 or 12?

File: exercises.md [hunk 3]
```diff
@@ -101,9 +107,10 @@ Briefly discuss what it means to be important as a group (if you want to)
 #### (10 mins) Together - Discuss
 
 Discuss in the group:
+
 * lines covered by many people?
 * lines named but not by a lot of people
-* Agree less than 8 of the most important lines
+* Agree less than 10 of the most important lines
```
Comment (danfishgold) [inline exercises.md]: (same 8 or 10 question)

**Review by danfishgold — COMMENTED** [review]
highlighting some inconsistencies I wasn't sure how to resolve

File: StarterKit/Session1/agenda.md [hunk 4]
```diff
@@ -78,7 +81,7 @@ Instantiation
 
 ## Content
 
-The goal of this exercise is to start to think about which lines in the code define it's essence, have the biggest impact or need to be paid close attention to.
+The goal of this exercise is to start to think about which lines in the code define its essence, have the biggest impact or need to be paid close attention to.
 
 ### (10 mins) Independently - Identify most important lines
```
Comment (danfishgold) [inline StarterKit/Session1/agenda.md]: I think 10 minutes might be too much for this

File: StarterKit/Session2/agenda.md (hunk 2 above)
Comment (katjam) [reply StarterKit/Session2/agenda.md]: I can't decide. Definitely 12 for the 1st session. After that 10 might be ok. It's also down to the length of the code. In some ways it's a shame ot waste more than 10 mins.

File: exercises.md (hunk 3 above)
Comment (katjam) [reply exercises.md]: Again - depends on the length of the code, so facilitator decides what is difficult but not impossible. I'd go with 8.

File: StarterKit/Session1/agenda.md (hunk 4 above)
Comment (katjam) [reply StarterKit/Session1/agenda.md]: Change to 8? Sometimes the code is harder to parse than others, but stick at 8.

Maybe we should make a general guidance about the facilitator using the timings as approx to aim for.

File: StarterKit/Session4-and-beyond/README.md [hunk 5]
```diff
@@ -1,8 +1,12 @@
 # You've run 3 sessions, now you're a pro!
 
-## Design your agenda with selected [exercises](https://github.com/CodeReadingClubs/Resources/blob/trunk/exercises.md). Keep practicing with the Basic Exercises and when you feel ready, try out some of the Advanced Exercises on a piece of code you're familiar with from a previous session.
+## Design your agenda with selected [exercises](https://github.com/CodeReadingClubs/Resources/blob/trunk/exercises.md)
```
Comment (katjam) [inline StarterKit/Session4-and-beyond/README.md:3]: These probably should not be headings. I think I did that in error.

File: StarterKit/Session4-and-beyond/README.md [hunk 6]
```diff
@@ -1,8 +1,12 @@
...
 
-## Grab some code from our [suggested code snippets](https://https://github.com/CodeReadingClubs/Resources/blob/trunk/StarterKit/Session4-and-beyond/code-snippets) or find your own. There is no bad choice - even html can be a conversation starter.
+Keep practicing with the Basic Exercises and when you feel ready, try out some of the Advanced Exercises on a piece of code you're familiar with from a previous session.
+
+## Grab some code from our [suggested code snippets](https://https://github.com/CodeReadingClubs/Resources/blob/trunk/StarterKit/Session4-and-beyond/code-snippets) or find your own
+
+There is no bad choice - even html can be a conversation starter.
```
Comment (katjam) [inline StarterKit/Session4-and-beyond/README.md:9]: :+1:

**Review by katjam — APPROVED** [review]
So many great changes. I'm not too worried by the fiddly timings. We'll have time to fix that in the weeks to come before too many people try to use.

My spelling/ grammar is terrible. Thanks so much!

**Review by katjam — APPROVED** [review]
Yes, let's get this in. :medal_sports:

**Pull request merged**